Libs/MRML/Core/vtkMRMLSliceNode.h

```diff
@@ -418,6 +440,11 @@ class VTK_MRML_EXPORT vtkMRMLSliceNode : public vtkMRMLAbstractViewNode
   vtkMatrix4x4 *UVWToSlice;
   vtkMatrix4x4 *UVWToRAS;
 
+  std::map<vtkStdString, vtkMatrix4x4*> OrientationMatrices;
+
+  vtkSmartPointer<vtkMatrix4x4> axialSliceToRAS;
```

Comment:
There is no need for these member variables. You just need to create matrices temporarily and set it in a preset,